cmd/cluster/core/dump.go

```diff
@@ -487,6 +488,7 @@ func DumpGuestCluster(ctx context.Context, log logr.Logger, kubeconfig string, d
 		// https://github.com/openshift/api/blob/2bde012f248a5172dcde2f7104caf0726cf6d93a/config/v1/types_cluster_version.go#L266-L270
 		&snapshotv1.VolumeSnapshotClass{},
 		&snapshotv1.VolumeSnapshotContent{},
+		&admissionregistrationv1beta1.ValidatingAdmissionPolicy{},
```

Comment:
You may also want to consider including `ValidatingAdmissionPolicyBinding` resources as well